deploy/helm/kuberhealthy/templates/deployment.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "kuberhealthy.name" . }}
+  labels:
+    app: {{ template "kuberhealthy.name" . }}
+    chart: {{ .Chart.Name }}
+spec:
+  replicas: {{ .Values.deployment.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "kuberhealthy.name" . }}
+  strategy:
+    rollingUpdate:
+      maxSurge: {{ .Values.deployment.maxSurge }}
+      maxUnavailable: {{ .Values.deployment.maxUnavailable }}
+    type: RollingUpdate
+  template:
+    metadata:
+      {{- if .Values.deployment.podAnnotations }}
+      annotations:
+      {{- range $key, $value := .Values.deployment.podAnnotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+      {{- end }}
+      {{- if .Values.prometheus.enabled -}}
+      {{- if .Values.prometheus.enableScraping -}}
+      {{- if not .Values.deployment.podAnnotations }}
+      annotations:
+      {{- end }}
+        prometheus.io/scrape: "true"
```

Comment:
Prometheus scrape annotations are also on the service -- this might be unnecessary to scrape pods here.